.github/workflows/main.yml

```diff
@@ -10,6 +10,7 @@ jobs:
   test:
     name: Test / OS ${{ matrix.os }} / Node ${{ matrix.node }} / SDK ${{ matrix.sdk }}
     strategy:
+      fail-fast: false
```

Comment:
Please enlighten me here, but why was this added?

Reply:
The test failed only on Dart dev, because of formatting, but I don't have the Dart dev version installed. Now it doesn't abort the tests on other channels, when one channel fails.

Reply:
Ah, nice! I wasn't aware of this flag, but it feels like we should have it for all the libraries huh?

Reply:
I think so. It's easier to track why it fails on server, but not locally.